appwrite/website
# Add blog post and assets for 10 Git commands you should start using today

## What does this PR do?
Adds blog post with assets for 10 new Git commands you should start using today, and updates cover/content for related git post

## Test Plan

- Run the app and navigate to `/10-git-commands-you-should-start-using`
- Confirm that blog post and assets are displayed correctly
